docs/element-definitions.md

```diff
@@ -367,21 +418,20 @@ The example below defines an Element matching only the null value.
 ### Boolean Element
 
 Type with domain of two values: true and false.
-`content` property MUST contain a boolean value if set.
 
 #### Template
 - `element` - `"boolean"`
 - `attributes`
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
-    - `fixed` ([String](#string-element)) - domain restricted to value given in `content`
+    - `fixed` ([String](#string-element)) - The type this Element describes is restricted to the value given in `content`
   - `validation` - _reserved for future use_
-  - `samples` ([Array[Boolean]](#array-element)) - array of alternative sample values for this Element; type of items in `samples` MUST match the type of this Element
-  - `default` (Boolean Element) - default value for this Element; type of `default` MUST match the type of this Element
-- `content` - ⊥ (false) or ⊤ (true)
+  - `samples` ([Array[[Boolean](#boolean-element)]](#array-element)) - Alternative sample values for this Element; type of items in `samples` MUST match the type this Element describes
```

Comment:
Bad linking. Use something similar to `([Array](#array-element)[[String](#string-element)])`